### PR TITLE
Add canBeCancelled property to Mollie_API_Object_Payment

### PR DESCRIPTION
**Pull request by insit**
Added this property to the Mollie_API_Object_Payment class to keep it in line with the docs.

### CODE REVIEW

File: src/Mollie/API/Object/Payment.php [hunk 1]
```diff
@@ -258,6 +258,13 @@ class Mollie_API_Object_Payment
 	 */
 	public $links;
 
+    /**
+     * Whether or not the payment can be cancelled.
+     *
+     * @var bool|null
+     */
```
Comment (Smitsel) [inline src/Mollie/API/Object/Payment.php]: Thanks for the PR, one small change request.
Could u make sure the indenting is right on this new docblock?

Comment (insit) [reply src/Mollie/API/Object/Payment.php]: Done :)

**Comment by Smitsel** [comment]
Thanks again! We love consistency ;)

**Comment by insit** [comment]
No problem at all! Could you maybe increase the version number in order for Composer to fetch a new version?

Kind regards!!

**Comment by willemstuursma** [comment]
We'll release a new version tomorrow.

**Comment by insit** [comment]
Thanks for releasing a new version. We are a very happy customer!

Kind regards!

Tom

**Pull request merged**